Formula/c/composer-lts.rb

```diff
@@ -0,0 +1,74 @@
+class ComposerLts < Formula
+  desc "Dependency Manager for PHP (LTS version)"
+  homepage "https://getcomposer.org/"
+  url "https://getcomposer.org/download/2.2.22/composer.phar"
+  sha256 "7d3500cc8c9a74b47e14103de150ac95c25ca227b39ffc89cb3a8b495b5db1d2"
+  license "MIT"
+
+  livecheck do
+    url "https://getcomposer.org/download/"
+    regex(%r{href=.*?/v?(\d+(?:\.\d+)+)/composer\.phar}i)
+  end
+
+  depends_on "php"
```

Comment:
According to the download page it doesn't work with php, only with disabled PHP versions (up to 7.1)

Reply:
It means that the LTS version also supports outdated versions of PHP.  
Please check the [source](https://github.com/composer/composer/blob/2.2/composer.json#L25).